packages/react-components/src/shared/tests/typescript/arrayify.ts

```diff
@@ -0,0 +1,16 @@
+import { arrayify } from "../../src";
+import { expectAssignable } from "../../../../../../typescript/tests/helpers";
+
+expectAssignable<number[]>(arrayify(1));
+expectAssignable<number[]>(arrayify([1]));
+
+// @ts-expect-error
+expectAssignable<number[]>(arrayify(["1"]));
```

Comment:
Since there is a limitation for the "expectNotAssignable" tests, do we really want to write any?

I ended up writing only assignable tests in the "CSS in JS" PR and I do have great coverage.